source/tachyons/_custom.css

```diff
@@ -37,3 +42,11 @@ a {
 .transition-300 {
   transition: 0.3s ease
 }
+
+h1, h2, h3, h4, h5, h6 {
```

Comment:
@ivanmauricio probably better to reference them as css vars, might be worth making a separate var for `roboto slab` with a couple of fallbacks (like Arial and Helvetica)